File: while3.c
// WAP, to enter a number then display factorial of that number
#include <stdio.h>
void main()
{
    int num;
    printf("Enter a number :");
    scanf("%d", &num);
    int fact = 1;
    int i = num;
    while (i > 0)
    {
        fact = fact * i;
        i--;
    }

    printf("%d", fact);
}

File: SEM 2 PPS/while3.py
# wap, to check wether the entered number is pallindrome or not using whilenum = int(input("Enter a number: "))
num = int(input("Enter a number: "))
num2 = num
sum = 0
rev = 0
while num > 0:
    digit = num % 10
    rev = rev*10+digit
    num //= 10

if num2 == rev:
    print("Number is pallindrome")
else:
    print("Number is not pallindrome")

# Output
# Enter a number: 121
# Number is pallindrome
